Release of Version 1.0

koflin opened the pull request:
Release of the first stable version of the VaorraAPI

koflin committed:
Basic Setup

-configured basic setup
-started specific and general storable fetching

koflin committed:
API now working

-first stable version of the VaorraAPI

koflin committed:
Merge branch 'feature/basic' into develop

StrawbrryFlurry commented on index.ts:
```diff
@@ -0,0 +1,133 @@
+    let user = req.headers["x-access-user"];
+    let token = req.headers["x-access-token"];
+
+    user = user ? user.toString() : undefined;
```
Send responsestatus in user variable definition [Line 24/25]

koflin replied:
Rather not do that, we'd need returns then. I'll just .toString() in the third if statements.

koflin committed:
Optimization and Versioning

-changed version
-added .js files to gitignore
-optimized an undefined check
-added a start-save command which automatically installs node_modules

koflin committed:
Ts ignores for missing files

-added ts ignores for missing sensitive files

koflin committed:
Typo fix

koflin committed:
Improved user-friendliness

-changed some responses
-added a response when providing an id which doesn't exist

koflin committed:
Updated deprecated functions

koflin committed:
Fixes and additions

-added /ids endpoint
-ids are getting parsed to a 24byte hex format for the database to proccess

koflin committed:
Improved create return

-object creation without an id now returns the autogenerated id from MongoDB
